lerna.json

```diff
@@ -2,5 +2,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.136.0"
+  "version": "0.136.2"
```

Comment:
Isn't this done via the Github action?

Reply:
I know you messaged on slack, but just to answer here by running lerna publish we can publish our feature + packages to npm. It tags the codebase thereby triggering our azure artefact builds and we can deploy a feature branch to an azure environment.

When the feature is ultimately merged into master all the versioning stuff will be overwritten